system-tests/project-fixtures/angular/cypress.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  component: {
+    devServer: {
+      framework: 'angular',
+      bundler: 'webpack',
+      webpackConfig: {
+        resolve: {
+          alias: {
+            '@angular/common': require.resolve('@angular/common'),
```

Comment:
Same issue we've had with other framework system tests. These aliases are not required when using an external application